app/admin/homepage.rb

```diff
@@ -0,0 +1,182 @@
+ActiveAdmin.register Homepage do
+  permit_params :internal_title, :section_title_one, :section_title_two, :section_title_three, :published, homepage_features_attributes: [:id, :title, :description, :url, :cta_text, :featured_image, :featured_image_file_name, :featured_image_content_type, :featured_image_updated_at, :featured_image_file_size, :image_alt_text, :delete_image, :section_id, :homepage_id, :position, :_destroy]
+
+
+  config.filters = false
+  config.sort_order = 'published_asc'
```

Comment:
just fyi with this set to `asc` the published homepage will be at the bottom of the list